### PR TITLE
Update prices_tokens.sql

### PR DESCRIPTION
**Pull request by Soho09**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -1619,6 +1619,7 @@ VALUES
     ("xen-xen-crypto", "ethereum", "XEN", "0x06450dee7fd2fb8e39061434babcfc05599a6fb8", 18),
     ("bob-bob", "ethereum", "BOB", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),
     ("unbnk-unbanked", "ethereum", "UNBNK", "0x06b884e60794ce02aafab13791b59a2e6a07442f", 18),
+    ("fin-definer", "ethereum", "FIN", "0x054f76beed60ab6dbeb23502178c52d6c5debe40", 18),
```
Comment (Hosuke) [inline models/prices/prices_tokens.sql:1622]: It seems `fin-definer` not existed on coinpaprika api:
https://api.coinpaprika.com/v1/coins/fin-definer

**Review by Hosuke — CHANGES_REQUESTED** [review]
Hi @Soho09, please check the id(which need to be available on coinpaprika) for the FIN token

File: models/prices/prices_tokens.sql (hunk 1 above)
Comment (Soho09) [reply models/prices/prices_tokens.sql:1622]: Please try again with capital “D” on coinpaprika

Comment (Soho09) [reply models/prices/prices_tokens.sql:1622]: https://coinpaprika.com/coin/definer-fin/

Comment (Hosuke) [reply models/prices/prices_tokens.sql:1622]: Do you have a specific link?

Comment (Soho09) [reply models/prices/prices_tokens.sql:1622]: Could you check this one?

Comment (Soho09) [reply models/prices/prices_tokens.sql]: https://coinpaprika.com/coin/definer-fin/

**Review by Hosuke — APPROVED** [review]
LGTM.✅
API: https://api.coinpaprika.com/v1/coins/fin-fin
SCAN: https://etherscan.io/token/0x054f76beed60ab6dbeb23502178c52d6c5debe40

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**